sessions/js-array-methods/demo-end/css/styles.css

```
@@ -0,0 +1,22 @@
+*,
+*::before,
+*::after {
+	box-sizing: border-box;
+}
+
+body {
+	margin: 0;
+	font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Helvetica, Arial, sans-serif,
+		'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol';
+}
+
+h1,
+h2 {
+	color: orangered;
+}
+
+article {
+	margin: 2rem;
+	padding: 0.5rem;
+	border: 1px solid black;
+}
```

```suggestion
:root {
  --nemo: #ff4a11;
  --granite: #252629;
  --water: #f3f5f9;
  --foam: #ffffff;
}

*,
*::before,
*::after {
  box-sizing: border-box;
}

body {
  margin: 0;
  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Helvetica,
    Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol';
}

h1,
h2 {
  color: var(--nemo);
}

article {
  margin: 2rem;
  padding: 0.5rem;
  border-radius: 10px;
  background: var(--water);
}
```
An alternative styling _suggestion_ featuring neuefische colors/color names. :)